.gitignore

```diff
@@ -18,3 +18,5 @@ spec/reports
 test/tmp
 test/version_tmp
 tmp
+.idea/
+.byebug_history
```

Comment:
let's drop these from this PR. They can be added to the `.git/info/exclude` for individuals that use these tools.